src/go_quic_server_session.cc

```diff
@@ -28,7 +28,7 @@ GoQuicServerSession::~GoQuicServerSession() {
 
 void GoQuicServerSession::InitializeSession(
     const QuicCryptoServerConfig& crypto_config) {
-  QuicSession::InitializeSession();
+  QuicSession::Initialize();
   crypto_stream_.reset(CreateQuicCryptoServerStream(crypto_config));
```

Comment:
This should be done before `QuicSpdySession::Initialize()` since the initialization logic changed on the underlying QUIC code.